magento/pwa-studio
# Remove references to window

## Description

In preparation to support SSR in a Node environment, we'll need to remove references to `window` and other browser-specific global objects that don't exist in Node.

Additionally it was attempted to move to an SSR friendly webpack loader that would work with CSS Modules on the server, but was discovered injection of CSS by effects could cause a FOUC in some areas. Until SSR is ready, we'll continue using `style-loader`, but begin the process of migrating to a `useStyle` hook that will eventually be hooked up to this new loader.

## Related Issue
PWA-1392

## Acceptance

### Verification Stakeholders

### Specification

### Verification Steps
1. General smoke test regression for client
2. Verify all form elements look and behave the same (several were re-factored to functional components)
3. `yarn build` and verify no errors
4. `cd packages/venia-concept/dist && node client.<hash>.SERVER.js` - verify no exceptions or errors thrown

## Screenshots / Screen Captures (if appropriate)

## Checklist
* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
